.vscode/settings.json

```diff
@@ -39,7 +39,7 @@
   "explorer.fileNesting.enabled": true,
   "explorer.fileNesting.patterns": {
     ".eslintrc.json": ".*",
-    "/webpages/settings/*.html": "/webpages/settings/${capture}.html"
+    "*.html": "${capture}.*"
```

Comment:
Can you send a screenshot of example files that are grouped?
I remember considering grouping our pseudo-vue components like this, and discarding the idea because the UI/UX wasn't good enough.

Reply:
@thecolaber I didn't do it that way because then it'd nest all the files in the popup directories together which makes people have to open up another level of nesting.

Reply:
That's a consistency issue. Popups and webpages should have the same type of folder system.

Reply:
So the current structure is:
```
webpages
- popups
  - cloud-games
    - index.html
    - index.js
    - index.css
  - scratch-messaging
   - index.html
   - index.js
   - index.css
```

We should change it to
```
webpages
- popups
  - cloud-games.html
  - cloud-games.js
  - cloud-games.css
  - scratch-messaging.html
  - scratch-messaging.js
  - scratch-messaging.css
```

Reply:
Popups are independent of each other, just like addons; 👎🏼 to this.

Reply:
Where would `popups/scratch-messaging/api.js` go?